xdc-plugin/tests/counter_golden.json

```diff
@@ -0,0 +1,33 @@
+{
+  "OBUF_6": {
+    "DRIVE": "I12",
```

Comment:
Shouldn't drive be a number (e.g. "DRIVE": 12)?

Reply:
Ah, I see the problem.  The XDC is wrong is this case, https://github.com/SymbiFlow/yosys-symbiflow-plugins/blob/bb93b7368f885447ee690c44383e4aa13aa0840a/xdc-plugin/tests/counter.xdc#L2 should be `set_property DRIVE 12` not `set_property DRIVE I12`

Reply:
Yepp, fixed.